src/screens/walletConnect/WalletConnect2Context.tsx

```diff
@@ -116,14 +116,15 @@ export const WalletConnect2Provider = ({
     PendingSession | undefined
   >(undefined)
   const [error, setError] = useState<WalletConnect2ContextArguments['error']>()
+  const [web3wallet, setWeb3Wallet] = useState<Web3Wallet | null>(null)
 
   const onSessionProposal = async (
     proposal: Web3WalletTypes.SessionProposal,
-    web3wallet: Web3Wallet,
+    _web3wallet: Web3Wallet,
```

Comment:
This naming is confusing, can you rename one of these two variables? perhaps `usersWallet`?

Reply:
underscore usually indicated internal variable, which is internal to the function, I don't think it's confusing but I can change if necessary

Reply:
Please, check again. I've changed the name